packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -2247,6 +2253,9 @@ export function attach(
       // Can view component source location.
       canViewSource,
 
+      // Can force the component to re-render.
+      canForceRerender,
```

Comment:
I think this should be conditional based on the element type. Right now it shows up for all types, including e.g. context providers and consumers, etc- re-rendering these doesn't make sense. Seems like this should be for class, function, forward ref, and _maybe_ memo components types only?